hcx/hcxinstallprep.md

```diff
@@ -0,0 +1,48 @@
+# HCX Installation Preparation
+
+## System Requirements
+
+### Virtual Hardware Requirements for HCX Appliances
```

Comment:
Why are we copying and pasting VMware documentation?  This will be hard to maintain as they change their docs.

Reply:
Makes sense. I will update it.